cirq/ops/common_gates.py

```diff
@@ -492,10 +492,10 @@ def _has_stabilizer_effect_(self) -> Optional[bool]:
     def _circuit_diagram_info_(self, args: 'cirq.CircuitDiagramInfoArgs'
                               ) -> Union[str, 'protocols.CircuitDiagramInfo']:
         if self._global_shift == -0.5:
-            return _rads_func_symbol(
-                'Rz',
-                args,
-                self._diagram_exponent(args, ignore_global_phase=False))
+            exponent = self._diagram_exponent(args, ignore_global_phase=False)
+            pi = sympy.pi if protocols.is_parameterized(exponent) else np.pi
+            angle_str = args.format_radians(radians=exponent * pi)
+            return f'Rz({angle_str})'
```

Comment:
We have this code copied three times in this file.  Is it worth writing a small helper function?

Reply:
Done.